pkg/yurtmanager/webhook/util/configuration/configuration.go

```diff
@@ -31,6 +31,12 @@ import (
 	webhookutil "github.com/openyurtio/openyurt/pkg/yurtmanager/webhook/util"
```

Comment:
I think this file should not be modified.

Reply:
fix